packages/model/src/standard/resources/namespace.resource.ts

```diff
@@ -10,5 +10,131 @@ import { Resource } from "#models/Resource.js";
 
 Resource.add({
     tag: "datatype", name: "namespace", description: "Namespace", xref: "core§7.19.2.43",
-    details: "The Namespace type identifies the namespace used for a semantic tag."
+    details: "The Namespace type identifies the namespace used for a semantic tag.",
+
+    children: [
```

Comment:
I think this should come out now, right?

Reply:
Thats generated yes, but seems also sometimes not 100% right (see Position)